app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -86,7 +86,14 @@ module(HELP_CONTENTS, [])
         <p>If you want to start from scratch, select "None".</p>
         <p>You can always edit the cluster configuration after you've created it.</p>`,
     'pipeline.config.expectedArtifact.matchArtifact': `
-        <p>This specifies which fields in your incoming artifact to match against. Every field that you supply will be used to match against all incoming artifacts. If all fields specified match, the incoming artifact is bound to your pipeline context.</p>
+        <p>
+          This specifies which fields in your incoming artifact to match against. If the artifact comes
+          from a pubsub message and a jinja template is specified for transforming those messages, then the fields
+          considered here are those populated by the jinja transformation.
+        </p>
+        <p>
+          Every field that you supply will be used to match against all incoming artifacts. If all fields specified match, the incoming artifact is bound to your pipeline context.
```

Comment:
s/fields specified/specified fields

Reply:
I would consider making this last sentence the first sentence of the help text. It makes the most sense to me.